app/lib/components/term.js

```diff
@@ -149,6 +160,10 @@ export default class Term extends Component {
     if (this.props.customCSS !== nextProps.customCSS) {
       this.term.prefs_.set('user-css', this.getStylesheet(nextProps.customCSS));
     }
+
+    if (this.devicePixelRatio !== window.devicePixelRatio) {
+      this.term.prefs_.set('font-smoothing', this.getFontSmoothing());
```

Comment:
Electron has a [BrowserWindow event for move](https://github.com/electron/electron/blob/master/docs/api/browser-window.md#event-move) that would be triggered when switching between screens.

Or just poll the window position:

```js
let position;
setInterval( () => {
  let next = window.screenLeft+','+window.screenTop;
  if (position && next!==position) {
    console.log('moved');
  }
  position = next;
}, 500);
```